docs/docsite/rst/community/communication.rst

```diff
@@ -134,6 +137,19 @@ Meetings on chat
 
 The Ansible community holds regular meetings on various topics on Matrix/IRC, and anyone who is interested is invited to participate. For more information about Ansible meetings, consult the `meeting schedule and agenda page <https://github.com/ansible/community/blob/master/meetings/README.md>`_.
 
+Ansible Community Topics
+========================
+
+The `Ansible Community Steering Committee <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst>`_ uses the `community-topics repository <https://github.com/ansible-community/community-topics/issues>`_ to asynchronously discuss with the Community and vote on Community topics in corresponding issues.
+
+Create a new issue in the `repository <https://github.com/ansible-community/community-topics/issues>`_ if you want to discuss an idea that impacts any of the following:
+
+* Ansible Community
+* Community collection best practices and `requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_
+* `Community collection inclusion policy <https://github.com/ansible-collections/ansible-inclusion/blob/main/README.md>`_
+* `The Community governance <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst>`_
+* Other proposals of importance that need the Committee's attention
```

Comment:
```suggestion
* Other proposals of importance that need the Committee or overall Ansible community attention
```

Reply:
I worry that limiting community-topics to just what the steering committee needs to hear would stifle conversation (like most if not all the docs-based discussion topics I've gotten great help with recently).

Reply:
indeed, thanks! I'll replicate this change to the SC docs